internal/integration/rest/utils.go

```diff
@@ -17,10 +17,16 @@
 }
 
 // GetFilterForProjects get filter string for projects list
-func GetFilterForProjects(fromDate, teamName, projectIds string) string {
+func GetFilterForProjects(fromDate, teamName, projectIds string, isDefaultProjectActiveSince bool) string {
```

Comment:
Please try making fromDate empty instead of passing information about the CLI arguments default state